pkg/api/platformapi/proxyapi/filteredgroupapi/update.go

```diff
@@ -0,0 +1,75 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package filteredgroupapi
+
+import (
+	"context"
+	"errors"
+
+	"github.com/elastic/cloud-sdk-go/pkg/api"
+	"github.com/elastic/cloud-sdk-go/pkg/client/platform_infrastructure"
+	"github.com/elastic/cloud-sdk-go/pkg/models"
+	"github.com/elastic/cloud-sdk-go/pkg/multierror"
+)
+
+var errVersionCannotBeLesserThanZero = errors.New("version cannot be less than 0")
+
+// UpdateParams is the set of parameters required for updating proxies filtered group
+type UpdateParams struct {
+	CreateParams
```

Comment:
I thought we decided against embedding the params since it leads to lots of headaches on error messages, and etc. Additionally, the multi error returned might be confusing?

Reply:
whoops! that one slipped in, will correct :)